src/hooks/Orderbook/useDrawOrderbook.ts

```diff
@@ -152,19 +166,34 @@ export const useDrawOrderbook = ({
     ctx.fill();
   };
 
+  const drawMineCircle = ({ ctx, rekt }: { ctx: CanvasRenderingContext2D; rekt: Rekt }) => {
+    const padding = 15;
+    ctx.beginPath();
+    ctx.arc(rekt.x1 + padding, (rekt.y1 + rekt.y2) / 2, 4, 0, 2 * Math.PI);
+    ctx.fillStyle = '#7774FF';
```

Comment:
nit: use `theme` var from `useAppThemeAndColorModeContext` hook.

I know its slightly trivial because the purple doesn't really change between themes.